config/rubocop/.rubocop.yml

```diff
@@ -510,9 +510,7 @@ Style/CommandLiteral:
   Enabled: false
 
 Style/CommentAnnotation:
-  Description: >-
-                 Checks formatting of special comments
-                 (TODO, FIXME, OPTIMIZE, HACK, REVIEW).
+  Description: 'Checks for presence of maintenance comments.'
```

Comment:
Nit: It's not checking for presence. It's checking for formatting of those comments when present.